torch/_dynamo/utils.py

```diff
@@ -140,6 +145,17 @@ def fmt_fn(values, item_fn=lambda x: x):
 }
 
 
+@contextmanager
+def disable_current_modes():
+    mode_len = _len_torch_dispatch_stack()
+    old_modes = [_pop_mode() for _ in range(mode_len)]
+    try:
+        yield old_modes
+    finally:
+        for i in range(mode_len):
```

Comment:
Should this be reversed?

Reply:
Nevermind, I see you are reversing with indexing below.  `reversed()`, might be more readable.